spec/recipes_spec/books/marketing/expected_output.xhtml

```diff
@@ -2845,35 +2845,35 @@
           <div data-type="question-solution" data-solution-source="collaborator" data-solution-type="detailed" id="auto_26033155-e4b9-468c-89d2-1a510ad1c431_263668-solution">
             <a class="os-number" href="#auto_26033155-e4b9-468c-89d2-1a510ad1c431_263668">1</a>
             <span class="os-divider">. </span>
-            <div class="os-solution-container"><span>c.</span>
+            <div class="os-solution-container"><span>c</span>
```

Comment:
So the dot option didn't change anything-- and a talk with Larissa says that the missing dots might have to do with the injected exercises themselves, or at least can be a new card to troubleshoot on later

Reply:
shouldn't be an issue with exercises themselves, since re-running these specs doesn't ping the exercises backend... investigating more

Reply:
found it 🤪 [this line in the recipe](https://github.com/openstax/cookbook/blob/main/lib/recipes/marketing/bake#L62) needs to be changed from `{options: {add_dot: true}}` to just `{add_dot: true}`

(this will reflect the fact that `question_options` no longer contains a hash of all arguments -- the previous behavior -- but now corresponds just to the `options` parameter of `BakeInjectedExerciseQuestion`.)